alkira/data_source_alkira_oci_vcn_connector.go

```diff
@@ -0,0 +1,35 @@
+package alkira
+
+import (
+	"github.com/alkiranet/alkira-client-go/alkira"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceAlkiraOciVcnConnector() *schema.Resource {
+	return &schema.Resource{
+		Description: "Use this data source to get an existing connector group.",
```

Comment:
connector group? This is connector_oci_vcn

Reply:
updated!